Google.PowerShell/Container/GkeClusterCmdlets.cs

```diff
@@ -580,4 +581,381 @@ protected override void ProcessRecord()
+        [ValidatePattern("[a-zA-Z][a-zA-Z0-9-]*[a-zA-Z0-9]")]
+        public string ClusterName { get; set; }
+
+        /// <para type="description">
+        /// The description of the cluster.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public string Description { get; set; }
+
+        /// <para type="description">
+        /// The number of nodes to create in the cluster.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        [ValidateRange(0, int.MaxValue)]
+        public int? InitialNodeCount { get; set; }
+
+        /// <para type="description">
+        /// The credential to access the master endpoint.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        [ValidateNotNull]
+        public PSCredential MasterCredential { get; set; }
+
+        /// <para type="description">
+        /// Stop the cluster from using Google Cloud Logging Service to write logs.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        public SwitchParameter DisableLoggingService { get; set; }
+
+        /// <para type="description">
+        /// Stop the cluster from using Google Cloud Monitoring service to write metrics.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        public SwitchParameter DisableMonitoringService { get; set; }
+
+        /// <para type="description">
+        /// Enables Kubernetes alpha features on the cluster. This includes alpha API groups
+        /// and features that may not be production ready in the kubernetes version of the master and nodes.
+        /// The cluster has no SLA for uptime and master/node upgrades are disabled.
+        /// Alpha enabled clusters are AUTOMATICALLY DELETED thirday days after creation.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        public SwitchParameter EnableKubernetesAlpha { get; set; }
+
+        /// <para type="description">
+        /// Name of the Google Compute Engine network to which the cluster is connected.
+        /// If left unspecified, the default network will be used.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public string Network { get; set; }
+
+        /// <para type="description">
+        /// The name of the Google Compute Engine subnetwork to which the cluster is connected.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public string Subnetwork { get; set; }
+
+        /// <para type="description">
+        /// The IP address range of the container pods in this cluster, in CIDR notation.
+        /// </para>
+        [Parameter(Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public string ClusterIpv4AddressRange { get; set; }
+
+        /// <para type="description">
+        /// The zones (in addition to the zone specified by -Zone parameter) in which
```

Comment:
It's because there is a primary zone that is associated with the cluster. Gcloud command also has 2 parameters, zone and additional-zone.